ui/pages/Project/components/RnaSeqOutliers.jsx

```diff
@@ -53,11 +53,13 @@ class RnaSeqOutliersGraph extends React.PureComponent {
       .append('g')
 
     dataPoints.append('circle')
-      .attr('cx', d => x(d[xField]))
-      .attr('cy', d => y(d[yField]))
-      .attr('r', 3)
-      .style('fill', 'None')
+      .attr('cx', d => x(d.zScore))
+      .attr('cy', d => y(d.pValue))
+      .attr('r', d => (d.deltaPsi === undefined ? 3 : r(Math.abs(d.deltaPsi))))
```

Comment:
Unfortunately this is actually a merge conflict and it is changing the behavior (using hardcoded `zScore` and `pValue` instead of `xField` and `yField`, checking `deltaPsi` which is no longer a supported field)